src/Containers.jsx

```diff
@@ -85,24 +85,31 @@ class Containers extends React.Component {
 
         if (force)
             args.timeout = 0;
-        utils.podmanCall("RestartContainer", args)
+        utils.podmanCall("RestartContainer", args, container.isRoot)
                 .catch(ex => this.setState({
                     actionError: cockpit.format(_("Failed to restart container $0"), container.names),
                     actionErrorDetail: ex.parameters && ex.parameters.reason
                 }));
     }
 
     renderRow(containersStats, container) {
-        const containerStats = containersStats[container.id];
+        const containerStats = containersStats[container.id + container.isRoot.toString()];
         const isRunning = container.status == "running";
         const image = container.image;
 
+        let proc = "";
+        let mem = "";
+        if (containerStats) {
+            proc = containerStats.cpu ? utils.format_cpu_percent(containerStats.cpu * 100) : <abbr title={_("not available")}>{_("n/a")}</abbr>;
+            mem = containerStats.mem_usage ? utils.format_memory_and_limit(containerStats.mem_usage, containerStats.mem_limit) : <abbr title={_("not available")}>{_("n/a")}</abbr>;
```

Comment:
Actually, these are columns in a table. IMHO it wouln't be too bad to just keep them empty, but I don't have a strong opinion.

Reply:
> Are stats not available for user containers in general? Or when would that happen?

When you run it on system without CGroupsV2

>  IMHO it wouln't be too bad to just keep them empty, but I don't have a strong opinion.

I also don't have a strong option about it, but I think it is better to show there something. Mostly because when these values are not there it indicates that that container is not running and this could confuse users and I can see those reports coming in that sometimes we don't show usage.